tests/cancun/eip6780_selfdestruct/test_selfdestruct_revert.py

```diff
@@ -292,41 +290,27 @@ def test_selfdestruct_created_in_same_tx_with_revert(  # noqa SC200
         )
         post[selfdestruct_recipient_address] = Account.NONEXISTENT  # type: ignore
 
-    nonce = count()
     tx = Transaction(
-        ty=0x0,
         value=0,
         data=entry_code,
-        chain_id=0x0,
-        nonce=next(nonce),
+        sender=sender,
         to=None,
-        gas_limit=100_000_000,
-        gas_price=10,
-        protected=False,
+        gas_limit=20_000_000,
     )
 
     state_test(env=env, pre=pre, post=post, tx=tx)
 
 
-@pytest.fixture
-def pre_with_selfdestructable(  # noqa: SC200
-    recursive_revert_contract_address: Address,
-    recursive_revert_contract_code: Bytecode,
-    selfdestruct_with_transfer_initcode_copy_from_address: Address,
-    selfdestruct_with_transfer_contract_initcode: Bytecode,
-    selfdestruct_with_transfer_contract_address: Address,
-    yul: YulCompiler,
-) -> Dict[Address, Account]:
-    """Preset for selfdestruct_not_created_in_same_tx_with_revert."""
-    return {
-        TestAddress: Account(balance=100_000_000_000_000_000_000),
-        selfdestruct_with_transfer_initcode_copy_from_address: Account(
-            code=selfdestruct_with_transfer_contract_initcode
-        ),
-        recursive_revert_contract_address: Account(code=recursive_revert_contract_code, balance=2),
-    }
-
-
+@pytest.mark.parametrize(
+    "recursive_revert_contract_address_init_balance",
+    [2],
+    ids=["init_balance_2"],
+)
+@pytest.mark.parametrize(
+    "same_tx",
+    [False],
```

Comment:
True?

Reply:
`test_selfdestruct_not_created_in_same_tx_with_revert`, following the docs string 
```
Same test as selfdestruct_created_in_same_tx_with_revert except selfdestructable contract
    is pre-existing.
```
so we should deploy this contract firstly.

Reply:
@winsvega I added the implementing for reentrancy module too.